docs/architecture/smart-rollups.mdx

```diff
@@ -1,266 +1,171 @@
+- The genesis commitment that forms the basis for commitments that rollups nodes publish in the future
+
+After it is originated, anyone can run a Smart Rollup node based on this information.
+
+### Commitment periods
+
+Starting from the rollup origination level, levels are partitioned into _commitment periods_ of 60 consecutive layer 1 blocks.
+During each commitment period, the each rollup node receives the messages in the rollup inbox, processes them, and updates its state.
+
+Because Smart Rollup nodes behave in a deterministic manner, their states should all be the same if they have processed the same inbox messages with the same kernel starting from the same origination level.
+This state is referred to as the "state of the rollup."
+
+At the end of the commitment period, each rollup node must publish a hash of its state to layer 1, which is called its _commitment_.
+Each commitment builds on the previous commitment, and so on, back to the genesis commitment from when the Smart Rollup was originated.
+
+Nodes must stake 10,000 tez along with their commitments.
+When nodes make identical commitments, their stakes are combined into a single stake for the commitment.
+
+Then a refutation period for the commitment period begins, and a new commitment period begins.
+
+### Refutation periods
+
+Because the PVM is deterministic and all of the inputs are the same for all nodes, any honest node that runs the same Smart Rollup produces the same commitment.
+As long as nodes publish matching commitments, they continue running normally.
+
+During the refutation period for a commitment period, if two or more nodes publish different commitments, two of them play a _refutation game_ to identify the correct commitment.
+The nodes automatically play the refutation game by stepping through their logic to identify the point at which they differ.
+At this point, layer 1 uses the PVM to identify the correct commitment.
+Then the protocol gives half of the incorrect commitment's stake to the correct commitment's stake, and burns the other half.
+The protocol eliminates the incorrect commitment because it has no stake.
+
+This refutation game happens as many times as is necessary to eliminate incorrect commitments.
+Because the node that ran the PVM correctly is guaranteed to win the refutation game, a single honest node is enough to ensure that the Smart Rollup is running correctly.
+This kind of Smart Rollup is called a Smart Optimistic Rollup because the commitments are assumed to be correct until they are proven wrong by an honest rollup node.
+
+When there is only one commitment left, either because all nodes published identical commitments or the refutation game eliminated all incorrect commitments, this correct commitment is said to be _cemented_ and therefore final and unchangeable.
+A commitment can't be cemented until the end of the refutation period and until all previous commitments are cemented.
```

Comment:
1. This phrase and the one above are true, but the same things can be said in a simpler way by combining them in one phrase (note that a refutation game can go past the end of the refutation period; it must just *start* before this time):
2. "final and unchangeable" are not consequences of the *definition* of "cemented" (so you cannot use "therefore"); you should either make them part of the definition or introduce it as synonyms
"When there is only one commitment left, either because all nodes published identical commitments and the refutation period expired, or because the refutation game eliminated all incorrect commitments, this correct commitment is said to be _cemented_ and becomes final and unchangeable.

Reply:
> When there is only one commitment left, either because all nodes published identical commitments or the refutation game eliminated all incorrect commitments, this correct commitment is said to be _cemented_ and becomes final and unchangeable. A commitment can't be cemented all previous commitments are cemented. The commitments for the next commitment period build on this cemented commitment.

Reply:
In fact, I'm not sure the first phrase is true. Cementation is not automatic, it's a L1 operation that must be injected by someone. Secondly, there may be several refutation games. Thirdly, we have to mention the refutation period has ended in the first case. And let's also remove the invalid sentence "A commitment can't be cemented all previous commitments are cemented.". This gives:

> When there is only one commitment left, either because all nodes published identical commitments during the whole refutation period or because this commitment won the refutation game(s) and eliminated all other  commitments, then this correct commitment can be cemented by a dedicated L1 operation and becomes final and unchangeable. The commitments for the next commitment period build on this cemented commitment.

Reply:
Updated. I didn't know that there was a specific layer 1 operation required to cement a commitment.